README.md

```diff
@@ -1,7 +1,13 @@
-# Tauri + React + Typescript
+# ACE v3
+The third iteration of the Advanced Cockpit Emulator.
 
-This template should help get you started developing with Tauri, React and Typescript in Vite.
+## Installation and usage
+1. [Install Rust and toolchain](https://tauri.app/v1/guides/getting-started/prerequisites)
+2. Clone the repo and `cd` into it
+3. `npm i`
+4. `corepack enable` - Tauri wants `yarn`, `corepack` helps to manage all the different package managers. Comes with Node 16.10+.
+5. `npm run tauri dev`
```

Comment:
How about change package.json, in 7th line, 

`"dev": "vite",` to `"dev": "tauri dev",`

looks vite is useless for me

Reply:
"dev" command is used under the hood, apparently. Added it as start so it can be used just as `npm start`.